src/Nethermind/Nethermind.Network/P2P/Subprotocols/Les/StatusMessage.cs

```diff
@@ -0,0 +1,84 @@
+﻿//  Copyright (c) 2018 Demerzel Solutions Limited
+//  This file is part of the Nethermind library.
+// 
+//  The Nethermind library is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU Lesser General Public License as published by
+//  the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+// 
+//  The Nethermind library is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+//  GNU Lesser General Public License for more details.
+// 
+//  You should have received a copy of the GNU Lesser General Public License
+//  along with the Nethermind. If not, see <http://www.gnu.org/licenses/>.
+
+using System;
+using Nethermind.Core.Crypto;
+using Nethermind.Dirichlet.Numerics;
+
+namespace Nethermind.Network.P2P.Subprotocols.Les
+{
+    public class StatusMessage : P2PMessage
+    {
+
+        public static class KeyNames
+        {
+            public const string ProtocolVersion = "protocolVersion";
```

Comment:
I will try to find why we need these

Reply:
I used them in the serializer, they're the keys that correspond with the data.
After my first typo, I decided it might be best to use constants, rather than string literals.

Though, maybe you'd prefer if they were defined in the serializer rather than here?